.github/workflows/ci.yml

```
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby:
+          - "3.2"
+          - "3.3"
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+
+      - name: Lint (Rubocop)
+        run: bundle exec rubocop
+
+      - name: Lint (Syntax Tree)
+        run: bundle exec stree check Gemfile $(git ls-files '*.rb') $(git ls-files '*.rake') $(git ls-files '*.thor')
+
+      - name: Build assets
+        run: |
+          cd messageformat-miniracer
+          yarn install && yarn webpack
```

dunno if actually shorter, or more concise, or anything but I prefer using `working-directory` 😄 

```suggestion
        working-directory: ./messageformat-miniracer
        run: yarn install && yarn webpack
```